src/librustdoc/html/render/mod.rs

```diff
@@ -2249,18 +2297,23 @@ fn sidebar_deref_methods(
                 .collect::<Vec<_>>();
             if !ret.is_empty() {
                 let id = if let Some(target_def_id) = real_target.def_id(c) {
-                    cx.deref_id_map.get(&target_def_id).expect("Deref section without derived id")
+                    Cow::Owned(
+                        cx.deref_id_map
+                            .get(&target_def_id)
+                            .expect("Deref section without derived id")
+                            .to_string(),
+                    )
                 } else {
-                    "deref-methods"
+                    Cow::Borrowed("deref-methods")
                 };
                 let title = format!(
-                    "Methods from {}&lt;Target={}&gt;",
+                    "Methods from {:#}<Target={:#}>",
                     Escape(&format!("{:#}", impl_.inner_impl().trait_.as_ref().unwrap().print(cx))),
-                    Escape(&format!("{:#}", real_target.print(cx))),
+                    real_target.print(cx),
```

Comment:
Why does this no longer escape the second interpolation here (the value of `Target=`)? And why does the first interpolation here get the debug format (`{:#}`)?